### ISSUE TITLE
XForms: Attribute groups with similar labels will override each other

### ISSUE THREAD

**Issue by linzjax** [Cadasta/cadasta-platform#748]
### Steps to reproduce the error
![image](https://cloud.githubusercontent.com/assets/8128188/18971191/201b876c-8662-11e6-8ed9-7c95aa87c69f.png)
If you have, say, two `party_attribute` groups with the same name, only the last one will remain.

[Please try to describe the actions that led to the incorrect behavior
in as much detail as possible.]

### Actual behavior
The notes attribute field will only contain notes from `party_attributes_individual`

### Expected behavior
The attributes should include notes from both `party_attribute_individual` and `party_attributes_default`

**Comment by linzjax** [comment]
Now that I think through this, is this a bug? If you're assuming that all default attributes are included in parties, why would you need a notes field in individual attributes as well?

Are there situations where attribute groups would contain the same fields?

**Comment by dpalomino** [comment]
Hi @linzjax. I don't think this is a bug, at least I cannot imagine a case where there would be a need for having the same labels in different groups.

**Comment by dpalomino** [comment]
This is not something we will be fixing now, field names should be different. 

Closing issue.
